packages/tpetra/core/test/FECrsMatrix/FECrsMatrix_UnitTests.cpp

```diff
@@ -275,17 +276,80 @@ TEUCHOS_UNIT_TEST_TEMPLATE_4_DECL( FECrsMatrix, Assemble1D, LO, GO, Scalar, Node
   TPETRA_GLOBAL_SUCCESS_CHECK(out,comm,success)
 }
 
+
+////
+TEUCHOS_UNIT_TEST_TEMPLATE_4_DECL( FECrsMatrix, Assemble1D_LocalIndex, LO, GO, Scalar, Node )
+{
+  typedef Tpetra::FECrsMatrix<Scalar,LO,GO,Node> FEMAT;
+  typedef Tpetra::CrsMatrix<Scalar,LO,GO,Node> CMAT;
+  typedef Tpetra::FECrsGraph<LO,GO,Node> FEG;
+
+  // get a comm
+  RCP<const Comm<int> > comm = getDefaultComm();
+  
+  // Generate a mesh
+  size_t numLocal = 10;
+  GraphPack<LO,GO,Node> pack;
+  generate_fem1d_graph(numLocal,comm,pack);
+  
+  // Make the graph    
+  // FIXME: We should be able to get away with 3 for StaticProfile here, but we need 4 since duplicates are
```

Comment:
Doesn't @tjfulle 's PR https://github.com/trilinos/Trilinos/pull/4733 fix this issue of resolving duplicates?